packages/Avatar/src/Avatar.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Avatar.styles";
+
+const propTypes = {
+  /** Avatar content. It can be initial as a string or icon */
+  children: PropTypes.node,
+  /** Background color of the Avatar */
+  backgroundColor: PropTypes.string.isRequired,
+  /** Color for the initial or icon */
+  color: PropTypes.string.isRequired,
+  /** Size of Avatar */
+  size: PropTypes.oneOf(ShirtSizes.LIMITED),
+};
+
+const defaultProps = {
+  children: null,
+  size: "medium",
+};
+
+function Avatar(props) {
+  const { backgroundColor, children, size, color } = props;
+
+  const avatarProps = {
```

Comment:
little confuse about this one, why do we need to split the props into two?

Reply:
was trying to replicate the format of other components.
Would this make more sense?
```
  const { backgroundColor, children, size, color } = props;

  const styleProps = {
    backgroundColor,
    size,
    color,
  };

    <sc.Avatar data-pka-anchor="avatar" {...styleProps}>
      {children}
    </sc.Avatar>
```
or `<sc.Avatar data-pka-anchor="avatar" {...props} />;`

Reply:
const { backgroundColor, color, ...moreProps  } = props;
```js
<sc.Avatar data-pka-anchor="avatar" $background={backgroundColor} $color={color}  {...moreProps} />
```

^ what about?

Reply:
Seems like `<div>` doesn't accept attribute since it's a container and can only be styled using css